src/datachain/lib/signal_schema.py

```diff
@@ -179,6 +179,16 @@ def from_column_types(col_types: dict[str, Any]) -> "SignalSchema":
                 )
         return SignalSchema(signals)
 
+    @staticmethod
+    def _get_bases(fr: type) -> list[tuple[str, str, Optional[str]]]:
+        bases: list[tuple[str, str, Optional[str]]] = []
+        for base in fr.__mro__:
```

Comment:
How will it fare if the type is somehow int or some basic types? 
<img width="677" alt="image" src="https://github.com/user-attachments/assets/19a38707-1003-41d9-9a2c-044ddc2e0cd1" />

Should we use getattr instead?

Reply:
Sorry for such a silly mistake 🤦